fvtest/compilertriltest/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set(CMAKE_CXX_EXTENSIONS OFF)
 
 add_compile_options(-pthread)
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fno-rtti -fno-threadsafe-statics -Wno-deprecated -Wno-enum-compare -Wno-invalid-offsetof -Wno-write-strings -O3 -fomit-frame-pointer -fasynchronous-unwind-tables -Wreturn-type -fno-dollars-in-identifiers -m64 -fno-strict-aliasing")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g -fno-threadsafe-statics -Wno-deprecated -Wno-enum-compare -Wno-invalid-offsetof -Wno-write-strings -O3 -fomit-frame-pointer -fasynchronous-unwind-tables -Wreturn-type -fno-dollars-in-identifiers -m64 -fno-strict-aliasing")
```

Comment:
This change seems like an important one. Can you explain why you need to remove the -fno-rtti flag to "support doing linkage definition on call sites in tril" ?

Reply:
Ah, whoops. I should remove that from this PR. 

That change makes Google Test's type-parameterized tests actually report the type they are testing, rather than just saying `<type>` all the time. 

But this work should be done as a different PR which I will start right away